iCubSheffield01/hardware/mechanicals/right_arm-eb4-j4_15-mec.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE params PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
 
-<params xmlns:xi="http://www.w3.org/2001/XInclude" robot="iCubSheffield01" build="1">
-     <group name="GENERAL">
-        <param name="MotioncontrolVersion">  4 </param>
+<params xmlns:xi="http://www.w3.org/2001/XInclude" robot="iCubGenova02" build="1">
```

Comment:
Leftover here :male_detective:

Reply:
@Nicogene: thanks a lot I missed that.
here is the commit which fixes: https://github.com/robotology/robots-configuration/commit/aaa797ac227cd24d709c5233d51a8efdf3dac3d2